packages/_debian-common/changelog

```diff
@@ -1,4 +1,4 @@
-libsuperiormysqlpp (0.3.3) UNRELEASED; urgency=medium
+libsuperiormysqlpp (0.4.0) UNRELEASED; urgency=medium
```

Comment:
I'm wondering, whether we should mention api breaking changes within changelog. Normally these changes would force major version update, but as we are still in version 0, this is a little bit different situation. But maybe explicit API changes warning could save someone headaches.